src/ssr-template-helper.js

```diff
@@ -55,7 +55,7 @@ export class SsrTemplateHelper {
    */
   isSupported() {
     const ampdoc = this.viewer_.getAmpDoc();
-    if (ampdoc.isSingleDoc()) {
+    if (ampdoc.isSingleDoc() && this.viewer_.isTrustedViewer()) {
```

Comment:
`isTrustedViewer` returns a promise, not a boolean.

Reply:
👍